process/block/baseProcess.go

```diff
@@ -2912,3 +2912,43 @@ func (bp *baseProcessor) revertGasForCrossShardDstMeMiniBlocks(added, pending []
 
 	bp.gasComputation.RevertIncomingMiniBlocks(miniBlockHashesToRevert)
 }
+
+// TODO: handle chain handler info on revert
```

Comment:
already done here https://github.com/multiversx/mx-chain-go/pull/7402

Reply:
if it is done, delete todo.